notifications_utils/recipients.py

```diff
@@ -572,13 +572,15 @@ def validate_phone_number(number, international=False):
     number = normalise_phone_number(number)
 
     if len(number) < 8:
-        raise InvalidPhoneError("Not enough digits")
+        raise InvalidPhoneError("Mobile phone number must be at least 8 digits long")
```

Comment:
@karlchillmaid I'm sure you considered this, so I'm just curious on the reasoning - what did you think of saying 'International mobile phone number' here - so that it's clearer to the user that we're validating it as a non-UK number?

Reply:
This is a good call, @samuelhwilliams. I’m going to do a quick review of all of these now I can see them in context.

Reply:
```suggestion
        raise InvalidPhoneError("This mobile number is too short")
```